src/v0/destinations/zendesk/transform.js

```diff
@@ -563,16 +564,17 @@ async function processTrack(message, destinationConfig, headers, baseEndpoint) {
     );
   }
 
+  const sourceName = getSourceName(destinationConfig);
   const eventObject = {
     description: message.event,
     type: message.event,
-    source: 'Rudder',
+    source: sourceName,
```

Comment:
Is it better to extract `sourceId` instead of sourceName which will be consistent to track any logs too ?

Reply:
can you link up the linear task to get the background of the issue ?